public/scripts/controllers/projectController.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
+  const projectController = {};
+
+  projectController.init = function() {
```

Comment:
you seem to be switching back and forth between about and project in this document